bugrap/src/main/java/org/vaadin/bugrap/DatabaseHelper.java

```diff
@@ -0,0 +1,14 @@
+package org.vaadin.bugrap;
+
+import org.vaadin.bugrap.domain.BugrapRepository;
+
+public class DatabaseHelper {
+	
+	public static final String DB_LOCATION = "/Users/onuridrisoglu/Development/eclipse-workspace/bugrap-data/bugrap";
+	
+	public static void main(String[] args) {
```

Comment:
As we are developing a web app, it would be better to get rid of the main method. You can find other ways of populating the database too.